### ISSUE TITLE
When the length is less than 24, it will cause checkMD to fail

### ISSUE THREAD

**Issue by magengbin** [ibm-messaging/mq-golang-jms20#54]
https://github.com/ibm-messaging/mq-golang-jms20/blob/044c73a84cae77c08601264527e412667d6ed0f6/mqjms/MessageImpl.go#L149

https://github.com/ibm-messaging/mq-golang/blob/b8f10b5addb40e6223f0e2d01a26a9c4377536bd/ibmmq/mqiMQMD.go#L114

Here you should use copy

```golang
	// Store the bytes form of the correlID
	copy(msg.mqmd.CorrelId, []byte(correlHexBytes))
```

**Comment by matrober-uk** [comment]
Hi @magengbin - thanks for highlighting this issue.  I'll look into reproducing the scenario in a test case and applying the fix you suggested (you're also very welcome to submit a testcase + fix in a PR yourself if you'd like, see the details in [Contributing](https://github.com/ibm-messaging/mq-golang-jms20#contributing)).

**Comment by matrober-uk** [comment]
Fixed under PR #57.
thanks for reporting the issue @magengbin!
